.github/workflows/scripts/check-version-consistency.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+import sys
+
+
+PACKAGE_ROOT = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+
+
+def release_branch_version_matches_setup_version(package_root):
+    os.chdir(package_root)
+    process = subprocess.run(["python", os.path.join(package_root, "setup.py"), "--version"], capture_output=True)
+    setup_version = process.stdout.strip().decode("utf8")
+
+    process = subprocess.run(["git", "branch", "--show-current"], capture_output=True)
+
+    full_branch_name = process.stdout.strip().decode("utf8")
+    branch_type, branch_name = full_branch_name.split("/")
+
+    if branch_type != "release":
+        error_message = f"The branch is not a release branch: {full_branch_name!r}"
+        print(error_message)
```

Comment:
print is disallowed by flake8 generally, but this might be the best way here.

Reply:
I originally didn't have the print statements but thought I'd add them in for explainability for developers looking at the logs. Do you think I should switch to logging?